internal/services/monitor/monitor_private_link_scope_resource.go

```diff
@@ -47,6 +48,30 @@ func resourceMonitorPrivateLinkScope() *pluginsdk.Resource {
 				ValidateFunc: validate.PrivateLinkScopeName,
 			},
 
+			"ingestion_access_mode": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(privatelinkscopesapis.AccessModeOpen),
```

Comment:
Could you confirm whether is this the default value returned by the API?

Reply:
That's correct. Open ingestion mode is the default.